dist/higlass-transcripts.js.LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+/*!
+ * The buffer module from node.js, for the browser.
+ *
+ * @author   Feross Aboukhadijeh <http://feross.org>
```

Comment:
This doesn't look right...

Reply:
Removed `dist` from the project and added it back to `.gitignore`.